docs/conf.py

```diff
@@ -14,12 +14,22 @@
 # Keep capitalization due to similar linking on GitHub's markdown preview.
 shutil.copy2('../CHANGELOG.md','./CHANGELOG.md')
 
+# for PDF output on Read the Docs
+project = "ROCm Documentation"
+author = "AMD"
```

Comment:
The author is AMD Inc. not just AMD

Reply:
Done in [e82fcbb](https://github.com/RadeonOpenCompute/ROCm/pull/2305/commits/e82fcbb107ad8254910d57f82abbb06613f5812d)

Do we also want to add a copyright?

Reply:
You should add that as well.

Reply:
[add copyright to pdf](https://github.com/RadeonOpenCompute/ROCm/pull/2305/commits/a3ec07e634775023bb537b307db72b2b8ee6e5a1)